src/60_Samples_%26_Templates/Product_Browse_Page.html

```diff
@@ -142,12 +156,74 @@
           <amp-img srcset="/img/ic_menu_white_1x_web_24dp.png 1x, /img/ic_menu_white_2x_web_24dp.png 2x" width="24" height="24" alt="navigation"></amp-img>
         </a>
         <input name="search" type="text" placeholder="Search" on="change:AMP.setState({
```

Comment:
Can you add a comment mentioning auto suggest and linking to the sample.

Reply:
Done!